src/client/opamCommands.ml

```diff
@@ -1324,6 +1333,15 @@ let reinstall =
     `S "OPTIONS";
     `S OpamArg.build_option_section;
   ] in
+  let assume_built =
+    Arg.(value & flag & info ["assume-built"]
+           ~doc:"For use on locally-pinned packages: assume they have already \
+                 been correctly built, and only run their installation \
+                 instructions, directly from their source directory. This \
+                 skips the build instructions and can be useful to install \
+                 packages that are being worked on. Implies $(i,--inplace-build). \
+                 No locally-pinned packages will be skipped.")
+  in
```

Comment:
Not sur about that, duplicated. Maybe a message redirecting to `opam install` manpage and its `--asume-built` option?

Reply:
Can be added as a top-level value in `OpamArgs` and used directly in the `Term.(pure foo $x $y $OpamArgs.assume_built)`